tv_apps/smart-system/fxa/js/screens/fxam_enter_email.js

```diff
@@ -196,6 +198,20 @@ var FxaModuleEnterEmail = (function() {
       }.bind(this)
     );
 
+    // There are 3 reasons why using setTimeout at this place:
+    // 1. Focus() only works in the setTimeout callback here
+    // 2. The input will be focused first and the keyboard will be brought
+    //    up. We need to do this after the slide up animation of the parent
+    //    fxa_dialog. But the fxa iframe has no way to know when the slide up
+    //    animation is finished.
+    // 3. Put the FxaModuleKeyNavigation.add in the onanimate callback in
```

Comment:
Can we add a fake animation? It's a shame to hard code a timeout and a potential unit test orange.